src/Microsoft.AspNetCore.Server.Kestrel.Core/KestrelServerLimits.cs

```diff
@@ -207,5 +210,26 @@ public TimeSpan RequestHeadersTimeout
                 _maxConcurrentUpgradedConnections = value;
             }
         }
+
+        /// <summary>
+        /// Gets or sets the default timeout period for receiving the request body. Defaults to 2 minutes.
+        /// </summary>
+        public TimeSpan DefaultRequestBodyTimeout
```

Comment:
Rename to just RequestBodyTimeout since it can be modified to a non-default value. I get that this value can be modified on a per-request basis, but that can be called out in the doc comments [like this](https://github.com/aspnet/KestrelHttpServer/blob/3dbfb003bb064e117b52d6336ee145cf602363ea/src/Microsoft.AspNetCore.Server.Kestrel.Core/KestrelServerLimits.cs#L145). This is better because it explains how to change the "default".